torch_frame/nn/models/trompt.py

```diff
@@ -122,7 +122,7 @@ def reset_parameters(self) -> None:
             trompt_conv.reset_parameters()
         self.trompt_decoder.reset_parameters()
 
-    def forward_stacked(self, tf: TensorFrame) -> Tensor:
```

Comment:
Let's make sure this change does not break the example code.

Reply:
Confirmed the change doesn't break these scripts across all supported task types:
```
examples/trompt.py
benchmark/data_frame_benchmark.py
benchmark/data_frame_text_benchmark.py
```